testing/src/test/proto/io/grpc/testing/helloworld.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016, Google Inc.
```

Comment:
There are a bunch of proto files for testing in the [`interop-testing`](https://github.com/grpc/grpc-java/tree/master/interop-testing/src/main/proto/io/grpc/testing/integration) directory. Can we reuse them in `test` project and maybe put them in some common directory?